### ISSUE TITLE
Disable automatic SSL/TLS setup

### ISSUE THREAD

**Issue by jeffalo** [mandatoryprogrammer/xsshunter-express#1]
It would be useful to be able to disable the automatic SSL/TLS so that you can handle the certificates seperately. 

For example, I already have an nginx reverse proxy setup with my cloudflare origin certificates. I don't need the automatic certificates and stuff, so if I could run it over http and handle the rest myself that would be useful.

**Comment by NHAS** [comment]
Yeah would be good, I also run it behind an nginx proxy, and having it try and acme to generate certs seems to be breaking the setup flow.

**Comment by 0xspade** [comment]
does the email notification works on you guys when you implemented this pull request ?

**Comment by wifiuk** [comment]
+1 being able to disabled SSL/TLS stuff would allow me to use it with cloudflare argo tunnels, at the moment I'm struggling to get it to work as I cant see where to disable the SSL auto setup.

**Comment by h3xitsec** [comment]
+1  i need to be able to run it behind my traefik proxy

**Comment by mux0x** [comment]
same for me

**Comment by adamjsturge** [comment]
@jeffalo @0xspade @NHAS @wifiuk @mux0x @h3xitsec 
I added this in my own fork a while ago but then after seeing how many out of date dependencies I made my own version in Golang. I'm using dependency bot and actively maintaining it https://github.com/adamjsturge/xsshunter-go

Feel free to check it out!
